protocols/kad/src/query/peers/closest.rs

```diff
@@ -211,16 +213,16 @@ impl ClosestPeersIter {
     /// If the iterator is finished, it is not currently waiting for a
```

Comment:
Can you document the case where `on_failure` returns `false`?

Reply:
Will do.